csrc/ops/alias.h

```diff
@@ -23,6 +23,8 @@ namespace nvfuser {
 TORCH_CUDA_CU_API Val* set(Val*);
 TORCH_CUDA_CU_API TensorView* set(TensorView*);
 
+TORCH_CUDA_CU_API TensorView* segment_alias(TensorView*);
```

Comment:
nit: it doesn't alias

Reply:
sorry about this one. Yeah I saw that as well I'll clean up the names and put in comments. for these.